### PR TITLE
GetFieldSetValues

### PR DESCRIPTION
**Pull request by adwhite75**
Takes in an object name and field set name and returns a text collection or CSV of fields in a field set

### CODE REVIEW

File: flow_action_components/GetFieldSetValues/force-app/main/default/classes/GetFieldSetValues.cls [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * @last modified on  : 11-02-2020
+ * @last modified by  : Adam White
+ * Modifications Log 
+ * Ver   Date         Author                               						  Modification
+ * 1.0   11-02-2020   Adam White, https://www.linkedin.com/in/declarativeninja/   Initial Version
+**/
+
+public with sharing class GetFieldSetValues {
+    @InvocableMethod
+    public static List <Results> getFieldSetVals(List<Requests> requestList) {
```
Comment (alexed1) [inline flow_action_components/GetFieldSetValues/force-app/main/default/classes/GetFieldSetValues.cls:14]: I've recently started using 'execute' as the name of this method, in each invocable action I build. It helps clarify that this method name isn't really used explicitly....

**Review by alexed1 — APPROVED** [review]
A nice solid addition. Go ahead and create a post for it.

**Comment by alexed1** [comment]
Note your project is missing the meta.xml files:
image
without those, the package won't build in sfdx

I suspect you did not create your apex classes from vscode sfdx menu items. That will autogenerate them for you:

image

**Comment by alexed1** [comment]
Got this error:
ERROR running force:package:version:create:  Apex Test Failure: Class.GetFieldSetValues.getFieldSetVals: line 26, column 1
Class.GetFieldSetValuesTests.testFieldSetValues: line 20, column 1 System.NullPointerException: Attempt to de-reference a null object

**Comment by adwhite75** [comment]
Hey @alexed1  - thanks for all the feedback! I actually need to include the sample field set in order for the test class to pass so that would explain the failures.

**Pull request merged**